doc/tor.md

```diff
@@ -11,12 +11,17 @@ configure Tor.
 The first step is running Bitcoin Core behind a Tor proxy. This will already anonymize all
 outgoing connections, but more is possible.
 
-	-proxy=ip:port  Set the proxy server. If SOCKS5 is selected (default), this proxy
+	-proxy=ip:port  Set a proxy server. If SOCKS5 is selected (default), this proxy
 	                server will be used to try to reach .onion addresses as well.
+	                You have to use -noonion or -onion=0 to explicitly disable
+	                outbound access to onion services.
 
 	-onion=ip:port  Set the proxy server to use for Tor onion services. You do not
-	                need to set this if it's the same as -proxy. You can use -noonion
+	                need to set this if it's the same as -proxy. You can use -onion=0
 	                to explicitly disable access to onion services.
```

Comment:
why changing from -noonion here?

Reply:
hmm, i must check my logs what -noonion did in comparison to -onion=0 its probably a tautology with the same effect.or there was an edge issue i forgot. I will check.

Reply:
@gmaxwell  tyi;  the diff was the log line
   ` Command-line arg: onion=false `   if ` -nonion`
vs `Command-line arg: onion="0"  `    if   `-onion=0  .`

So since we compare in code like this
```
if (onionArg == "0") { // Handle -noonion/-onion=0
```

I was wary that some compiler might not set onionArg to "0";  if
m_settings.command_line_options == false instead of "0"

I might be over cautious here?

Reply:
Arguments all uniformly support nofoo and foo=0 as being equivalent.